.eslintignore

```diff
@@ -12,3 +12,4 @@ jest.*config.js
 *.min.js
 /src/main/assets/js/jquery.js
 /src/main/assets/js/mojAll.js
+/src/main/assets/scss/home.scss
```

Comment:
With the file now being smaller, can we remove the exclusion + fix any linting issue if possible?

Reply:
sorted.